eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1472,6 +1472,49 @@ public function testSwapLocation()
         );
     }
 
+    /**
+     * Test for the swapLocation() method with custom aliases.
+     *
+     * @covers \eZ\Publish\API\Repository\LocationService::swapLocation
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testSwapLocationCustomAliases()
+    {
+        $repository = $this->getRepository();
+        $locationService = $repository->getLocationService();
+        $urlAliasService = $repository->getURLAliasService();
+        $this->createLanguage('pol-Pl', 'Polski');
```

Comment:
```suggestion
        $this->createLanguage('pol-PL', 'Polski');
```
We try to use the language code in this form, like `'eng-GB'`